docs/markdown/snippets/dia_sdk.md

```diff
@@ -0,0 +1,3 @@
+## Support for DIA SDK
+
+Added support for Windows Debug Interface Access SDK (DIA SDK) dependency. It allows reading with MSVC debugging information (.PDB format). This dependency can only be used on Windows, with msvc or clang-cl compiler.
```

Comment:
You mention msvc, clang and clang-cl in the documentation, but not clang here. Is it normal?

Reply:
Fixed.